src/main/java/frc/robot/constants/Constants2023.java

```diff
@@ -100,16 +102,6 @@ public class Constants2023 extends GameConstants {
     public static final double CLIMBER_BALANCE_THRESH=30;
     public static final double CLIMBER_TIMEOUT_S=10;
 
-    //Intake
-    public static final int INTAKE_MOTOR_1_ID = 111;
-    public static final int INTAKE_MOTOR_2_ID = 12;
-
-    public static final double INTAKE_MOTOR_1_SPEED = 1.00;
-    public static final double INTAKE_MOTOR_2_SPEED = 1.00;
-
-    public static final int INTAKE_MOTOR_PEAK_CURRENT_LIMIT = 0; //value has to be tweaked
-    public static final int INTAKE_MOTOR_PEAK_CURRENT_DURATION = 10; //value has to be tweaked
-    public static final int INTAKE_MOTOR_CONTINUOUS_CURRENT_LIMIT = 1; //value has to be tweaked
-    public static final boolean INTAKE_CURRENT_LIMIT_ENABLED = false; //value has to be tweaked
-
+    //Deployer Constants
+    public static final int DEPLOYER_MOTOR_ID = 11;
```

Comment:
You can't have two motor constants that have the same ID. Right now I can see that FEEDER_MOTOR_ID and DEPLOYER_MOTOR_ID have the same ID. You need to change FEEDER_MOTOR_ID to a different value, like 111.